packages/core/src/components/autosuggest/autosuggest.tsx

```diff
@@ -398,8 +414,17 @@ export class Autosuggest {
                   aria-selected={(suggestion === this.selectedSuggestion).toString()}
                   aria-label={suggestion.value}
                 >
-                  <span class="value">{this.markTerms(suggestion.value, terms)}</span>
-                  {suggestion.type ? <span class="type">{suggestion.type}</span> : undefined}
+                  <div>
+                    <span class="value">{this.markTerms(suggestion.value, terms)}</span>
+                    {suggestion.type ? <span class="type">{this.markTerms(suggestion.type, terms)}</span> : undefined}
+                  </div>
+                  {suggestion.extras &&
+                    this.getChunkedExtras(suggestion.extras).map((chunk) => (
+                      <div>
+                        {chunk[0] ? <span class="extra">{this.markTerms(chunk[0], terms)}</span> : undefined}
+                        {chunk[1] ? <span class="extra">{this.markTerms(chunk[1], terms)}</span> : undefined}
```

Comment:
Overwogen om te itereren?

```suggestion
                        {chunk.map(c => (<span class="extra">{this.markTerms(c, terms)}</span>}
```

Wat mij betreft leesbaarder, en leuke extra (🥁): De enige plek waar het aantal kolommen wordt bepaald is nu in `private getChunkedExtras()` met `Math.floor(index / 2)`.

Reply:
✅